src/index.js

```diff
@@ -670,7 +671,18 @@ const members = {
       return this.getAttribute('name');
     },
     set(name) {
-      return this.setAttribute('name', name);
+      const ret = this.__setAttribute('name', name);
```

Comment:
Looking at it, I don't think it will because you're using the double-underscored version, but checking just in case.